dist/shiny-server-client.js

```diff
@@ -1953,9 +1954,14 @@ exports.createFactory = function (options) {
       }
     });
 
-    var conn = new global.SockJS(url, null, {
-      protocols_whitelist: whitelist
-    });
+    var transportDebugging = options.transportDebugging && options.transportDebugging == true;
```

Comment:
Maybe just blindly pass it down if it is specified; let the sockjs library interpret its meaning.